commands/setup.go

```diff
@@ -70,11 +70,11 @@ var CommonFlags []cli.Flag = []cli.Flag{
 		EnvVars: []string{"DEALBOT_WALLET_ADDRESS"},
 	},
 	&cli.StringFlag{
-		Name:     "miner",
-		Usage:    "address of miner to make deal with",
-		Aliases:  []string{"m"},
-		EnvVars:  []string{"DEALBOT_MINER_ADDRESS"},
-		Required: true,
+		Name:    "miner",
+		Usage:   "address of miner to make deal with",
+		Aliases: []string{"m"},
+		EnvVars: []string{"DEALBOT_MINER_ADDRESS"},
+		//Required: true,
```

Comment:
This parameter is going to be needed for storage deals. Why is not required?

Reply:
I removed it, as `required` because I am not sure it should be specified with the `daemon` command. The `dealbot daemon` runs alongside a `lotus-miner`, and makes deals with MANY miners, whereas this option is for the CLI to specify ONE miner to make a deal with.